documentation/modules/configuring/proc-changing-topic-replicas.adoc

```diff
@@ -130,4 +130,12 @@ bin/kafka-topics.sh --bootstrap-server
 my-topic  Partition: 0  Leader: 0  Replicas: 0,1,2 Isr: 0,1,2
 my-topic  Partition: 1  Leader: 2  Replicas: 1,2,3 Isr: 1,2,3
 my-topic  Partition: 2  Leader: 3  Replicas: 2,3,4 Isr: 2,3,4
-----
\ No newline at end of file
+----
+
+. Finally, patch the `KafkaTopic` custom resource to match the new replication factor, and let the operator do the rest
+[source,shell,subs=+quotes]
+----
+kubectl patch kt my-topic --type merge -p '
+  spec:
+    replicas: 3'
+----
```

Comment:
I would just write it in a generic way:

```suggestion
. Finally, update the `KafkaTopic` resource to match the new replication factor (in `.spec.replicas`).
```

Many users use various tooling and cannot do a patch command like this.